### DISCUSSION TITLE
X265

### DISCUSSION THREAD

**Discussion by therealgrimshader** [Hubro/clipface#40] [Q&A] [answered]
I'm assuming mp4s in x265 aren't support? None of the x265 files i tried will play back video, audio works though.

**Comment by Hubro** [comment, marked as answer]
Clipface exposes the video unchanged to the browser using a HTML5 video tag.

In other words, which containers and codecs are supported are completely up to the browser. Unfortunately, browser support for H.265/HEVC is still really poor: https://caniuse.com/hevc

I would recommend using H.264 and AAC audio in a vanilla MP4 contaienr for Clipface. These seem to be best supported across most browsers. MKV containers also work well in my experience, but your mileage may vary.

**Reply by therealgrimshader** [reply]
Looks like that's the case, I was using this for posting video captured from phones (many capture in HEVC) to get round Discords file size limit.

Waiting on the WIP transcode option is probably my best bet for now.

Cheers for responding mate!
